### ISSUE TITLE
Twitter Validation is Failing

### ISSUE THREAD

**Issue by GuardianMajor** [sybrew/the-seo-framework#280]
Over the past two days I have noticed that while Google+ and Facebook shares works flawlessly and they pull the necessary metadata, Twitter is failing and the reason eludes me. I am suspecting a possible change in their API or something, not sure.

I have checked the source code and all the Twitter related metadata is present like always, nothing has changed there. So I decided to check using Twitter's Card Validator (https://cards-dev.twitter.com/validator) and it provides the following message in the log suggesting for some reason it is not finding the metadata it expects, which I cannot explain.

**Log**
```
INFO:  Page fetched successfully
WARN:  No metatags found
```

Are we aware of any changes that could be resulting in this? 

Looking through issues of this kind raised in the community, I came across suggestions that `content-type` is not being sent, so that could be causing the issue, someone suggested passing `Content-Type: text/html` will fix the issue but what I don't understand is why this would even be the case? 

Another suggested that 

> ... by replacing Unicode apostrophes in the meta tags (’) with regular straight ASCII apostrophes (') ...

fixed the issue but then again my source code does not show this to be the case. And another suggested that removing `x-content-type-options` was the solution for him but then again, doesn't seem to be the issue here for me.

I was wondering if you had any context on this? I can provide the header requests and responses for you on the validation page if it would help but didn't want to include them right now for sake of brevity. Thank you in advance.

**Comment by sybrew** [comment]
Hi @GuardianMajor 

The suggestions provided are for non-WordPress installations. WordPress should handle the content type, TSF should handle the correct output.

I suspect this might've been a hick-up in Twitter's API. Feel free to submit the page again in Twitter's validator today to see if anything has changed.
If the error comes back, please notify me of the page in question, and I'll investigate. You have my email address already, for if you wish to do this confidentially.

**Comment by GuardianMajor** [comment]
Good day my friend, yes, I suspected that as well and glad to hear you confirm it. I was pretty sure it wasn't related to WP sites, but wanted to include them for sake of completeness and to demonstrate that I did research it before bring it up.

It has had hiccups in the past but usually resolves nearly immediately or very shortly after. But this was over the span of a few days so it seems to have been a larger issue. However, the ones shared today seem to process and show up properly. I have gone back to test the ones that didn't show up before and it seems to have resolved as well.

Thank you again, and yes I have your private information and will contact you if need be, just wanted to limit the intrusion to you personally.
